### DISCUSSION TITLE
[C-X] Working Group Post Consortia Working Model -- Moving Meeting Time?

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/sig-release#12] [Polls]
Hi all,

at the current Timeslot i have an internal Refinement. 
Could we allign together on Wendsday Afternoon?

**Comment by alexschirra** [comment]
Hi, could we add an additional timeslot after 2:30 pm? Our ART Sync (BD RES) is scheduled every wednesday 1:00 -2:30 pm.

**Reply by FaGru3n** [reply]
Updated the Poll ;)

**Comment by FaGru3n** [comment]
Info @ Voters: 
Changing the poll question and/or options will reset the votes. Anyone who has already voted will need to submit their vote again.

--> Please vote again for your desired slot 👍🏼

**Comment by SAPTimmy** [comment]
Added my vote

**Comment by FaGru3n** [comment]
Hi all,

seems Wednesday 13-14 CET won, who can update this Meeting? 

or even should we close this disscussion because not needed?

**Comment by FaGru3n** [comment]
obsolete and will closed
